DeviceAPICharter.html

```diff
@@ -168,8 +168,10 @@ <h3 id="rec-track">Normative Specifications</h3>
 
 <h4>Others</h4>
 <dl>
-  <dt id="network"><a href="https://www.w3.org/TR/netinfo-api/">Network Information API</a></dt>
-  <dd>An API to discover the current network characteristics — the group will determine whether the existing implementations of this API on mobile warrants restarting its standardization process</dd>
+  <dt id="network">Network Information API</dt>
+  <dd>An API to discover the current network characteristics — the group will determine whether the existing implementations of this API on mobile warrants restarting its standardization process

```

Comment:
maybe indicate that the determination will be done in collaboration with the WICG

Reply:
Good suggestion, incorporated.